client/components/category-navigation/style.scss

```
@@ -0,0 +1,116 @@
+@import "@wordpress/base-styles/breakpoints.scss";
+@import "@automattic/components/src/styles/typography";
+@import "@automattic/typography/styles/variables";
+
+.category-navigation {
+	background: var(--studio-white);
+}
+
+.category-navigation-inner {
+	display: flex;
+}
+
+.category-navigation__buttons {
+	display: flex;
+	gap: 16px;
+	position: relative;
+	padding-right: 17px;
+	flex: none;
+
+	&::before {
+		content: "";
+		position: absolute;
+		right: 0;
+		top: 50%;
+		margin-top: -7px;
+		width: 1px;
+		height: 14px;
+		display: block;
+		border-radius: 2px;
+		background: #2c3338;
+	}
+}
+
+.category-navigation__list {
+	overflow: hidden;
+	position: relative;
+
+	.category-navigation__list-inner {
+		overflow: auto;
+		display: flex;
+		gap: 16px;
+
+		&::-webkit-scrollbar {
+			display: none; /* for browsers that support this */
+		}
+	}
+
+	.category-navigation__arrow {
+		width: 44px;
+		height: 44px;
+		position: absolute;
+		left: 0;
+		top: 0;
+		z-index: 2;
+		background: var(--studio-white) url(./images/arrow.svg) no-repeat center center;
+		transform: rotate(180deg);
+		cursor: pointer;
+
+		&:focus:not(:disabled) {
+			box-shadow: none;
+		}
+
+		&::before {
+			content: "";
+			position: absolute;
+			top: 0;
+			right: 100%;
+			width: 58px;
+			height: 44px;
+			background: linear-gradient(270deg, var(--studio-white) 40.38%, transparent 100%);
+			cursor: none;
+			pointer-events: none;
+		}
+
+		&.right {
+			left: auto;
+			right: 0;
+			transform: rotate(0);
+		}
+
+		@media ( max-width: $break-wide ) {
+			display: none;
+		}
+	}
+}
+
+.category-navigation__button {
+	display: flex;
+	flex: none;
```

```suggestion
	flex-shrink: 0;
```
I think this would communicate the intention more clearly.